glsk/glsk-document-cim/pom.xml

```diff
@@ -20,7 +20,7 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <!-- Dependencies versions -->
         <junit.version>4.13.1</junit.version>
-        <powsybl.core.version>4.4.0</powsybl.core.version>
+        <powsybl.core.version>4.6.0-SNAPSHOT</powsybl.core.version>
```

Comment:
I don't understand why you have to rewrite the version everywhere. Can you look at this other repositories to see how this is managed efficiently?

Reply:
Actually there is mismatch on the property name between the poms : powsyblcore on the parent -> powsybl.core on the children. That's why the property was not correctly propagated. Fixing it.